example/lib/develop/peripheral_list/peripheral_list_state.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:blemulator_example/develop/model/ble_peripheral.dart';
+
+class PeripheralListState {
+  final List<BlePeripheral> peripherals;
+  final bool scanningEnabled;
```

Comment:
does it mean that if the boolean has been changed then the list of the peripherals is redrawn?

Reply:
Fixed - added conditions to `BlocBuilder` calls.